packages/ozone-typescript-client/src/itemClient/itemClientImpl.ts

```diff
@@ -118,6 +128,25 @@ export class ItemClientImpl<T extends Item> implements ItemClient<T> {
 		return this.client.call<UUID[]>(request)
 	}
 
+	private createGraphQLClient(): ApolloClient<NormalizedCacheObject> {
+		const link = new HttpLink({
+			fetch: (uri, options) => {
+				const req = new Request(`${this.baseUrl}/rest/v3/graphql`, { method: options?.method, body: options?.body })
+				return this.client.call<string>(req).then(it => {
+					let obj = {}
+					// @ts-ignore
+					obj.text = () => new Promise(function(resolve) { resolve(JSON.stringify(it)) })
+					return obj as Response
```

Comment:
Couldn't this be slightly refactored to
```suggestion
					return {
						text: () => Promise.resolve(JSON.stringify(it))
					} as Response
```
?